data/wanderers start.txt

```diff
@@ -397,6 +463,11 @@ mission "Wanderers: Defend Vara Ke'sok"
 			label yes
 			`	The Wanderer leaves to join one of the ships that is taking off, and you quickly ready your own ship to follow...`
 				launch
+			label hint
+			`Touching down onto the planet, several Wanderers gather around your ship and prepare to carry the cargo out of your ship. Before the Wanderer you transported to <planet> can say a word, a shrill bird cry fills the air."`
+			`	Looking around, you suddenly see a flurry of activity as many of the Wanderers fly to their ships, which begin to take off from the planet. In the sky, you see bright flashes that may be distant weapons fire or explosions.`
+			`	The Wanderer that tasked you to carry the construction materials speaks up. "Unfettered [raider, pirate] fleet comes. We will put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
+				goto choice
```

Comment:
So that reading this in the file doesn't require backtracking, you should put lines 466-470 between lines 455 and 456, then have the `goto choice` be after line 455.

Reply:
Moved.